src/libraries/System.Private.CoreLib/src/System/Math.cs

```diff
@@ -1682,5 +1686,227 @@ internal static ulong ConvertToUInt64Checked(double value)
             ThrowHelper.ThrowOverflowException();
             return 0;
         }
+
+        [StackTraceHidden]
```

Comment:
It may be better to have this in a separate .cs that's only included on CoreCLR/NAOT and only on 32-bit architectures that need it.

Reply:
ILLink step was failing with missing definition.